packages/plugin-mparticle-react-native/lib/index.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars, class-methods-use-this, import/no-unresolved */
+import {
+  Event, Properties, RequestLoggerPlugin,
+} from '@itly/sdk';
+import MParticle from 'react-native-mparticle';
+
+// Native initialization: https://docs.mparticle.com/developers/sdk/react-native/getting-started/#installation
```

Comment:
Can you please add this to the README instead along with any other basic setup/install instructions.